rules/tines_rules/tines_story_items_destruction.py

```diff
@@ -0,0 +1,8 @@
+def rule(event):
+    return event.get('operation_name') == 'StoryItemsDestruction'
+
+def title(event):
+    operation = event.get('operation_name')
+    user = event.get('user_email')
+    tines_instance = event.get('p_source_label')
+    return f"Tines {operation} by {user} on {tines_instance}"
```

Comment:
This detection should use the global helper for tines to provide alert_context https://github.com/panther-labs/panther-analysis/blob/master/global_helpers/panther_tines_helpers.py#L1